### PR TITLE
3/Update the README.md file add description and clone information.

### PR DESCRIPTION
**Pull request by SarahLizDettloff**
Updated read me with:
1) Clone information
2) Screenshot
3) Contributors
4) About

In response of "Update the README.md file add description and clone information." on Issue #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
...
+* Bubble Sort
+* Optimized Bubble Sort
+* Merge Sort
+
+## Clone Directions
+``` git clone https://github.com/tanwanimohit/visualize-algorithms.git ```
+
+
+
+## Contrbutors
```
Comment (tanwanimohit) [inline README.md]: Remove Contributors info, as anyone can see that in insights.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,22 @@
-# visualize-algorithms
-:fire: Visualize Algorithms
+<h1 align="center">Visualize Algorithms</h1>
+ <br>
+ <img src="https://github.com/SarahLizDettloff/visualize-algorithms/blob/master/screenshot.png?raw=true" alt="Visualize Algorithms"> </br>
```
Comment (tanwanimohit) [inline README.md]: No need of screenshot !
Remove Screenshot image also.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,22 @@
...
+A tool which provides data visualization to algorithms. Current features:
+* Selection Sort
+* Bubble Sort
+* Optimized Bubble Sort
+* Merge Sort
+
+## Clone Directions
+``` git clone https://github.com/tanwanimohit/visualize-algorithms.git ```
+
+
```
Comment (tanwanimohit) [inline README.md]: Add How people can contribute.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,22 @@
-# visualize-algorithms
-:fire: Visualize Algorithms
+<h1 align="center">Visualize Algorithms</h1>
+ <br>
```
Comment (tanwanimohit) [inline README.md]: Remove This br tag as well.

File: README.md (hunk 1 above)
Comment (SarahLizDettloff) [reply README.md]: Contributors section has been removed. Please see newest commit.

File: README.md (hunk 2 above)
Comment (SarahLizDettloff) [reply README.md]: Image has been removed. Please see newest commit.

File: README.md (hunk 4 above)
Comment (SarahLizDettloff) [reply README.md]: Tags removed.

File: README.md (hunk 3 above)
Comment (SarahLizDettloff) [reply README.md]: 👍

**Pull request merged**